src/System.Security.Permissions/src/System/Diagnostics/EventLogPermissionEntryCollection.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+namespace System.Diagnostics
+{
+    [Serializable]
+    public class EventLogPermissionEntryCollection : CollectionBase
+    {
+        internal EventLogPermissionEntryCollection() { }
```

Comment:
@morganbr I meant given they are just stubs. Ok @juady please just delete the test so we can merge this.